lib/models/blueprint.js

```diff
@@ -781,9 +781,10 @@ let Blueprint = CoreObject.extend({
     @param {Function} destPath
     @param {Object} templateVariables
     @param {String} file
-    @return {FileInfo}
+    @param {Object} options may be used when buildFileInfo is customized in a blueprint
+    @return {FileInfo | null}
   */
-  buildFileInfo(intoDir, templateVariables, file) {
+  buildFileInfo(intoDir, templateVariables, file /*, options */) {
```

Comment:
@NullVoxPopuli if this is commented, is this actually needed?

Reply:
It's just how I document that an argument is passed to this function.
It's used in blueprints/app/index.js, and passed where buildFileInfo is called in this file